client/js/widgets/widget.js

```diff
@@ -1271,14 +1271,16 @@ export class Widget extends StateManaged {
       }
 
       if(a.func == 'MOVEXY') {
-        setDefaults(a, { count: 1, face: null, x: 0, y: 0, snapToGrid: true });
+        setDefaults(a, { count: 1, face: null, x: 0, y: 0, snapToGrid: true, resetOwner: true });
         if(this.isValidID(a.from, problems)) {
           await w(a.from, async source=>{
             for(const c of source.children().slice(0, a.count || 999999).reverse()) {
               if(a.face !== null && c.flip)
                 c.flip(a.face);
               await c.bringToFront();
               await c.setPosition(a.x, a.y, a.z || c.get('z'));
+              if(c.get('_ancestor') && widgets.get(c.get('_ancestor')).get('childrenPerOwner') && a.resetOwner)
```

Comment:
If it's a parameter, it should probably not check `childrenPerOwner`? I added that initially to keep the impact small if we don't do a file updater.

Reply:
I thought about that, but wasn't if you would think that is best.  Fixed.